### PR TITLE
Added Recipe Model w/ Neha and Muxuan

### PR DESCRIPTION
**Pull request by nkompella**
Created recipe model & relationships between recipe and user models

### CODE REVIEW

File: app/models/recipe.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 require 'net/http'
 class Recipe < ActiveRecord::Base
+
+    has_and_belongs_to_many :users
```
Comment (omidafshar) [inline app/models/recipe.rb:4]: Every user has multiple recipes, and every recipe can be associated with multiple users. This is a good, sensible design. Hopefully adding the concept of a "favorited" recipe will go off of this smoothly

File: db/migrate/20190423235125_create_join_table_users_recipes.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableUsersRecipes < ActiveRecord::Migration
+  def change
+    create_join_table :users, :recipes do |t|
```
Comment (omidafshar) [inline db/migrate/20190423235125_create_join_table_users_recipes.rb:3]: Does this just create an intermediate table that joins users and recipes?

Comment (nkompella) [reply db/migrate/20190423235125_create_join_table_users_recipes.rb:3]: Yes!

File: db/schema.rb [hunk 3]
```diff
@@ -11,11 +11,22 @@
...
   create_table "recipes", force: :cascade do |t|
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
+    t.integer  "day"
+    t.integer  "slot"
+    t.string   "title"
+    t.integer  "calories"
+    t.integer  "readyminutes"
```
Comment (cheolhojang) [inline db/schema.rb:23]: this may not show up for every single recipe

**Review by cheolhojang — COMMENTED** [review]
good creation

File: db/schema.rb [hunk 4]
```diff
@@ -11,11 +11,22 @@
...
+    t.integer  "price"
+  end
+
+  create_table "recipes_users", id: false, force: :cascade do |t|
```
Comment (omidafshar) [inline db/schema.rb:27]: So each entry of this table consists of user-recipe pairs

Comment (nkompella) [reply db/schema.rb:27]: Right

**Comment by omidafshar** [comment]
LGTM. No tests necessary for this since it's just adding a model

**Pull request merged**